_includes/parse-server/push-notifications-android.md

```diff
@@ -1,67 +1,48 @@
-#### Configure Broadcast Receiver and Permissions
+#### FCM Push Setup
 
-Add the following service and broadcast receiver definitions to `AndroidManifest.xml` immediately before the *closing* `</application>` tag:
+Add dependency to the application level `build.gradle` file.
 
-```xml
-<service android:name="com.parse.PushService" />
-<receiver android:name="com.parse.ParsePushBroadcastReceiver"
-android:exported="false">
-<intent-filter>
-<action android:name="com.parse.push.intent.RECEIVE" />
-<action android:name="com.parse.push.intent.DELETE" />
-<action android:name="com.parse.push.intent.OPEN" />
-</intent-filter>
-</receiver>
-<receiver android:name="com.parse.GcmBroadcastReceiver"
-android:permission="com.google.android.c2dm.permission.SEND">
-<intent-filter>
-<action android:name="com.google.android.c2dm.intent.RECEIVE" />
-<action android:name="com.google.android.c2dm.intent.REGISTRATION" />
-
-<!--
-IMPORTANT: Change "com.parse.starter" to match your app's package name.
--->
-<category android:name="com.parse.starter" />
-</intent-filter>
-</receiver>
-
-<!--
-IMPORTANT: Change "YOUR_SENDER_ID" to your GCM Sender Id.
--->
-<meta-data android:name="com.parse.push.gcm_sender_id"
-  android:value="id:YOUR_SENDER_ID" />;
+```groovy
+dependencies {
+    implementation 'com.parse:parse-android-fcm:latest.version.here'
```

Comment:
Is there something that would replace the 1.17.0 with the later version when we do a new release? Otherwise, I think it is probably pretty annoying to have to edit these files all the time when there is a new release. Maybe just making this always say `latest.version.here` would be easier to maintain?

Reply:
We can use `https://api.bintray.com/packages/parse/maven/com.parse:parse-android/images/download.svg` for our badge version..

Reply:
(Trying to figure that out took 10 mins!)

Reply:
Do you mean we would include the badge below that line of text or something?